src/schemas/profiles.py

```
@@ -1,13 +1,60 @@
 from datetime import date
+from typing import Optional
 
 from fastapi import UploadFile, Form, File, HTTPException
 from pydantic import BaseModel, field_validator, HttpUrl
 
-from validation import (
+from src.validation.profile import (
     validate_name,
     validate_image,
     validate_gender,
-    validate_birth_date
+    validate_birth_date,
+    validate_info
 )
 
-# Write your code here
+
+class ProfileRequestSchema(BaseModel):
+    first_name: str
+    last_name: str
+    gender: str
+    birth_date: str
+    info: str
+    avatar: UploadFile = File(...),
+
+    @field_validator("first_name", "last_name")
+    @classmethod
+    def _validate_name(cls, v: str):
+        return validate_name(v)
```

The validation functions like `validate_name` don't return a value; they raise an exception on failure. By returning `validate_name(v)`, you are setting the field's value to `None` (the implicit return value of the function).

The correct pattern is to call the validation function to let it raise an exception if needed, and then return the original value `v`:
```python
validate_name(v)
return v
```
This applies to all the validator methods in this schema.